storage/oasis/nodeapi/eden/node.go

```diff
@@ -161,6 +160,18 @@ func (c *ConsensusApiLite) RoothashEvents(ctx context.Context, height int64) ([]
 	return events, nil
 }
 
+func (c *ConsensusApiLite) RoothashLastRoundResults(ctx context.Context, height int64, runtimeID common.Namespace) (*roothash.RoundResults, error) {
+	// Hack: deserialize into nodeapi (Damask) structure.
```

Comment:
should rewrite this so that it's not even regarded as a hack. we're intentionally choosing to make the Damask APIs future compatible by adding fields when needed